OpenSim/Simulation/Test/testForces.cpp

```diff
@@ -1670,3 +1677,20 @@ void testExternalForce()
             ASSERT_EQUAL(def, val, 10*accuracy);
     }
 }
+
+void testSerializeDeserialize() {
```

Comment:
> So changing the model file will not make the test better.

My suggestion was to load an old-format model that contains both enabled and disabled muscles, and then check that the `appliesForce` property was set for each muscle as expected. It's not necessary to include this test (I don't think tests like this exist for other property changes), but it seems like that's what was intended here.